### PR TITLE
Add snyk tool into image

### PR DESCRIPTION
**Pull request by MartinBasti**

### CODE REVIEW

File: test/entrypoint.sh [hunk 1]
```diff
@@ -36,3 +36,4 @@ parse_hacbs_test_output sanity_label_check conftest unittest.json
 
 echo "Starting Integeration-Tests"
 bats $POLICY_PATH/conftest.sh
+exec $@
```
Comment (MartinBasti) [inline test/entrypoint.sh:39]: I don't understand purpose of this even after reading https://github.com/redhat-appstudio/hacbs-test/pull/20/files#r875791522

Comment (MartinBasti) [reply test/entrypoint.sh:39]: I got it now and I don't like it

Comment (MartinBasti) [reply test/entrypoint.sh]: https://github.com/redhat-appstudio/hacbs-test/pull/40/commits/904df2356b445415ff529a54f70481af5904961b I did something, please check if you like it

**Review by jsztuka — APPROVED** [review]
LGTM, I would just make sure that none process in worker image lifecycle is expecting entrypoint that is being removed.

**Review by Josh-Everett — APPROVED** [review]
LGTM see @jsztuka comment above

**Comment by MartinBasti** [comment]
I just added small test to check presence of working snyk binary in the image

**Pull request merged**